components/authentication/component-maintainer.yaml

```diff
@@ -1,7 +1,7 @@
 kind: ClusterRole
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
-  name: pipeline-maintainer
+  name: component-maintainer
```

Comment:
pipeline-maintainer is still used in authentication/e2e-tests.yaml and authentication/group-sync/group-sync-ci.yaml

Reply:
I missed that plus one more occurrence 5d1f710